compiler/natives/src/reflect/swapper.go

```diff
@@ -0,0 +1,29 @@
+// +build js
+
+package reflect
+
+func Swapper(slice interface{}) func(i, j int) {
```

Comment:
Yeah, this is to enable the new way of sorting in 1.8.

https://beta.golang.org/pkg/sort/#example_Slice